tools/advisor/advisor/db_options_parser.py

```diff
@@ -140,22 +165,20 @@ def load_from_source(self, options_path):
                     error = 'Not able to parse line in Options file.'
                     OptionsSpecParser.exit_with_parse_error(line, error)
 
-    def setup_misc_options(self, misc_options):
-        self.misc_options = {}
-        if misc_options:
-            for option_pair_str in misc_options:
-                option_name = option_pair_str.split('=')[0].strip()
-                option_value = option_pair_str.split('=')[1].strip()
-                self.misc_options[option_name] = option_value
-
     def get_misc_options(self):
+        # these are options that are not yet supported by the Rocksdb OPTIONS
+        # file, hence they are provided and stored separately
         return self.misc_options
 
     def get_column_families(self):
         return self.column_families
 
     def get_all_options(self):
+        # This method returns all the options that are stored in this object as
+        # a: Dict[<sec_type>.<option_name>: Dict[col_fam, option_value]]
         all_options = []
+        # Example: in the section header '[CFOptions "default"]' read from the
```

Comment:
How many section types we have? Can you point me to where it is documented?

Reply:
https://github.com/facebook/rocksdb/blob/17731a43a6e6a212097c1d83392f81d310ffe2fa/options/options_parser.h#L24

Reply:
Can you put inline comment like: "Refer to OptionSection for other type of section types."